Camera2Extensions/app/src/main/java/com/example/android/camera2/extensions/fragments/CameraFragment.kt

```diff
@@ -790,9 +806,11 @@ class CameraFragment : Fragment(), TextureView.SurfaceTextureListener {
     if (!jpegSupportedSizes.isEmpty()) {
       postviewSize = jpegSupportedSizes[0]
       postviewFormat = ImageFormat.JPEG
-    } else {
+    } else if (!yuvSupportedSizes.isEmpty()){
       postviewSize = yuvSupportedSizes[0]
       postviewFormat = ImageFormat.YUV_420_888
+    } else {
+      return null
```

Comment:
I'm wondering if we should also check for jpeg R supported sizes?

Reply:
This was to address an uncaught error in the sample app, but otherwise I opted for post view capture (jpeg or yuv) + still capture (jpeg_r) for simplicity